packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/About.2.doc.mdx

```diff
@@ -2,26 +2,119 @@ import { Notice } from "@cerner/terra-docs";
+* [Accessibility](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#accessibility)
+* [Implementation](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#implementation)
+* [Terra Standards](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#terra-standards)
+* [Properties](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#properties)
+
+## Overview
+The Worklist Data Grid is an [accessible grid](https://www.w3.org/WAI/ARIA/apg/patterns/grid/) component that allows the user to display data in a table like structure of rows and columns.
+Worklist Data Grids are ideal when you want to layout data in a way that allows for the user to navigate and interact with it while minimizing the need to navigate the user away from the grid.
+
+The Worklist Data Grid displays a header row and one or more rows of data.
+The component contains only one tab stop with the arrow keys used for primary navigation.
+See [keyboard interactions](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#keyboard-interactions) for all supported keys.
+Other key features of the component are listed out in the [Features](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#features) section.
+
+<DefaultWorklistDataGrid title='Default Worklist Data Grid' />
+
+  ### Features
+
+  * [Navigation](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/default-worklist-data-grid)
+  * [Masked Cells](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/default-worklist-data-grid)
+  * [Cell selection](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/cell-selection)
+  * [Row selection](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/row-selection)
+  * [Pinned columns](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/pinned-columns)
+  * [Column Sorting](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/worklist-data-grid-column-states)
+  * [Column Error States](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/worklist-data-grid-column-states)
+  * [Single Cell Navigation (Dive-In)](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/worklist-data-grid-focusable-cell)
+
+Refer to the [examples](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#examples) section for detailed demonstration of the features supported by the Worklist Data Grid.
+
+## Design Decisions
+The Worklist Data Grid supports the [grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/) as documented by the ARIA Authoring Practices Guide.
+
+The Worklist Data Grid:
+  * supports only one tab stop. Tabbing into the Worklist Data Grid will either go to the last visited cell, or the first cell upon initial entry.
```

Comment:
I'm wondering if we should be capitalizing the first word of each of these bullet points. I know they are sentence fragments, but IMO capitalization makes bulleted lists look cleaner and easier to scan quickly. I like the style detailed in [this article](https://www.instructionalsolutions.com/blog/bulleted-list-punctuation#:~:text=Yes.,format%20using%20semicolons%2C%20illustrated%20above.) for example.